x-pack/qa/core-rest-tests-with-security/src/yamlRestTest/resources/rest-api-spec/test/synonyms/10_synonyms.yml

```diff
@@ -0,0 +1,109 @@
+setup:
+  - skip:
+      features: headers
+      version: " - 8.9.99"
+      reason: Introduced in 8.10.0
+
+  - do:
+      security.put_user:
```

Comment:
I wasn't able to use a `roles.yml` file / user setting for `build.gradle`. I'm sure I'm missing something, in the meanwhile adding users and roles via security API worked.

If you can help with another approach, that would be much appreciated 🙏

Reply:
Done, thanks for the pointer!